sync/syncer.go

```diff
@@ -145,97 +148,175 @@ func (s *Syncer) Synchronise() {
 				return
 			}
 		}
-		s.ValidateLayer(lyr)
+
+		mu.Lock()
```

Comment:
doesn't this make the go routine redundant?

Reply:
allows fetching the next one while tortoise is running